src/Uno.UI.Composition/Composition/Visual.skia.cs

```diff
@@ -29,7 +71,6 @@ public Vector2 AnchorPoint
 		set
 		{
 			SetProperty(ref _anchorPoint, value);
-			Compositor.InvalidateRender(this);
```

Comment:
Is it expected to remove this ?

Reply:
It's an unnecessary call. SetProperty already calls `InvalidateRender` internally.